### ISSUE TITLE
Landing Page Announcement Text Update

### ISSUE THREAD

**Issue by kbwagner** [openopps/openopps-platform#2151]
Update the announcement text on the landing page to read: 
We’ve partnered with the Federal Acquisitions Institute (FAI) to launch the new Acquisitions Open Opportunities (NEEDS LINK) portal.  Starting May 18, 2018, you can search for opportunities within the Acquisitions career field.  Check out this video to learn more.

Interested in partnering with us?  Contact us at OpenOpps@usajobs.gov.

**Comment by bmcorum** [comment]
@margaretdumas and @Kibrahacha67 this is ready for testing.

**Comment by Kibrahacha67** [comment]
5-15-18 LHammond in testing

**Comment by kbwagner** [comment]
5/15/18 - Kwagner - updated AC with new requirement and sending back to development.

**Comment by kbwagner** [comment]
5/15/18 - KWagner Chrome
@jhanson12  Spelling error on "You can now search for opportunities within the Acquisitios career field."
Also - did you and Min decide to remove the "Announcement" header?  At this point that's not a big deal but the spelling error is.  Thanks!

**Comment by jhanson12** [comment]
Announcement is staying.

**Comment by kbwagner** [comment]
@jhanson12 Yes Min told me last night that you guys had agreed to it staying - thanks!

**Comment by bmcorum** [comment]
@margaretdumas and @Kibrahacha67 this is ready for testing.

**Comment by Kibrahacha67** [comment]
5-16-18 LHammond in testing
Testing completed in IE, Edge and Chrome.  Moving to review.

**Comment by kbwagner** [comment]
5/16/18 - KWagner tested in Chrome and mobile Safari.
@LisaLNelson ready for final review.

**Comment by LisaLNelson** [comment]
approved
